roles/wazuh-indexer/defaults/main.yml

```diff
@@ -1,13 +1,12 @@
 ---
 
 single_node: false
+generate_certs: true
 instances:
   aio:
     name: node-1
-    ip: '{{ hostvars[inventory_hostname].private_ip }}'
+    ip: "{{ hostvars.aio.private_ip }}"
```

Comment:
I would use the default value the components use for the `network.host`/ip value in their respective config files.